client/app/components/Form/__tests__/Form.spec.jsx

```diff
@@ -22,25 +19,23 @@ const getComponent = () => (
 );
 
 describe('Form', () => {
-  it('has no errors when submit is clicked', () => {
-    const wrapper = mount(getComponent());
-    act(() => wrapper.find('input[name="some-text-name"]').prop('onBlur')({
-      currentTarget: { value: 'Hello' },
-    }));
-    expect(wrapper.find('.error').length).toBe(0);
-    wrapper.find('input[type="submit"]').simulate('click');
-    expect(wrapper.find('.error').length).toBe(0);
+  const {
+    getByRole, getByText, queryByRole, getByPlaceholderText,
+  } = screen;
+  test('it renders properly', () => {
```

Comment:
Just for consistency's sake with the rest of the tests, let's use `it` instead of `test`. Let's also add a line between each test block for readability.

Reply:
Sounds good, I will make these changes. Thanks for looking everything over!